crm_lead_to_event_registration/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2016 Tecnativa - Jairo Llopis
+# Copyright 2016 Tecnativa - Pedro M. Baeza
+# Copyright 2017 Tecnativa - Vicent Cubells
+# Copyright 2018 Tecnativa - Cristina Martin
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Register a lead directly in an event",
+    "version": "11.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    "website": "https://www.tecnativa.com",
```

Comment:
@cristinamartinrod  IMHO Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)